tests/integration/targets/azure_rm_virtualmachinescaleset/tasks/main.yml

```
@@ -41,7 +43,9 @@
   azure_rm_loadbalancer:
     resource_group: "{{ resource_group }}"
     name: testLB1
-    public_ip_address_name: testPublicIP1
+    frontend_ip_configurations:
+      - name: frontendipconf0
+        public_ip_address: testPublicIP1
```

Add backend_address_pools. Before the deletion, backend_address_pools has a default value if it is not configured
```suggestion
        public_ip_address: testPublicIP1
      backend_address_pools:
        - name: backendaddrp0
```

---

just to confirm, both load balancers will use the same backend address pool?

---

Yes, you can define it differently. Thank you!

---

so I should use different backends for each one:

```
+    backend_address_pools:
+      - name: backendaddrp0
```

and 

```
+    backend_address_pools:
+      - name: backendaddrp1
```

I will run the integration tests again too just to make sure.